test/unit/requester-core.test.js

```diff
@@ -79,6 +79,47 @@ describe('requester util', function () {
             });
         });
 
+        it('should accept URL in uppercase', function () {
+            var request = new sdk.Request({
+                url: 'HTTP://POSTMAN-ECHO.COM/POST',
+                method: 'POST',
+                header: [{
+                    key: 'alpha',
+                    value: 'foo'
+                }],
+                body: {
+                    mode: 'raw',
+                    raw: '{"alpha": "foo"}'
+                }
+            });
+
+            expect(requesterCore.getRequestOptions(request, {})).to.eql({
```

Comment:
Avoid asserting on the entire object, only check for the part this test is meant for.